site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
+
+### Setting up your Mattermost instance and AWS for deploying custom apps
+
+Note that custom apps are not supported in the Mattermost Cloud - they're only used for developer testing convenience. Provisioning in the AWS cloud environment is done by the **appsctl** tool after completing the environment set up.
+
+We need an app bundle to provision an app. The bundle might be provisioned from the local disk, from S3 (not implemented yet), or from a URL (not implemented yet). Provisioning consists of three parts which will be completed via the **appsctl** tool:
```

Comment:
1/5 that features without a ticket that are not implemented should not be documented.

Reply:
Dylan and I talked through this also.  The wording remained from an early version of the doc, when we didn't have all the install methods in place. 

Since this doc is about deploying and provisioning via AWS, I removed the other portions for "local disk" and via "URL".  Those are install methods, but should be, if they aren't, described separately.

See. [`93fac70` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/93fac702af145c812e15e829a79194d818b07a2b)